build-fail-reminder.py

```diff
@@ -76,11 +77,51 @@ def jdefault(o):
 marked as maintainer for the above mentioned package (or project
 maintainer if the package has no explicit maintainer assigned)
 
+Kind regards,
+%(sender)s
+""",
+u"""Dear Package maintainers and hackers.
+
+Below package(s) in %(project)s have been failing to build for at
+least 4 weeks. We tried to send out notifications to the
+configured bugowner/maintainers of the package(s), but so far no
+fix has been submitted. This probably means that the
+maintainer/bugowner did not yet find the time to look into the
+matter and he/she would certainly appreciate help to get this
+sorted.
+
+""",
+u"""
+Unless somebody is stepping up and submitting fixes, the listed
+package(s) are going to be removed from %(project)s.
+
 Kind regards,
 %(sender)s
 """
 )
 
+
+def SendMail(logger, project, sender, to, fullname, subject, text):
```

Comment:
Perhaps try and use / improve:

https://github.com/openSUSE/openSUSE-release-tools/blob/0862a640433ea8c8d32a45caec40d3025c2bd799/osclib/util.py#L126-L154